runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/AbstractWindmillStream.java

```diff
@@ -149,7 +146,7 @@ private StreamObserver<RequestT> requestObserver() {
   }
 
   /** Send a request to the server. */
-  protected final void send(RequestT request) {
+  protected void send(RequestT request) {
```

Comment:
can this remain final? don't see new override

Reply:
missed it in the revert done.